### ISSUE TITLE
How to tell if using CPU or GPU + another random issue

### ISSUE THREAD

**Issue by mata3o** [neuralchen/SimSwap#74]
Hello team, love what you all have made here as its a very interesting concept! I would like to know how to tell if one is using CPU or GPU and if using CPU how to change to GPU processing.

My 2nd issue is with swapping any pictures, Any video swap works fine but when i try doing any picture swap i get this series of messages (read/write is enabled for everything):

#1  C:\Users\User1\MiniConda3\envs\simswap\lib\site-packages\torch\serialization.py:656: SourceChangeWarning: source code of class 'torch.nn.modules.dropout.Dropout' has changed. Tried to save a patch, but couldn't create a writable file Dropout.patch. Make sure it doesn't exist and your working directory is writable.

#2  C:\Users\User1\MiniConda3\envs\simswap\lib\site-packages\torch\nn\functional.py:3455: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired.

#3  C:\Users\Matt\MiniConda3\envs\simswap\lib\site-packages\torch\nn\functional.py:3500: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True.

Any tips or help would be greatly appreciated and my apologies if i didn't post these issues right as its my first time doing all this

**Comment by woctezuma** [comment]
These are warnings, not errors, so it is fine imo.

**Comment by mata3o** [comment]
@woctezuma There is no picture output in the specified location (output folder for me) but videos end up in there no problem. Its got me proper stumped! Appreciate the response

**Comment by simbiant** [comment]
@mata3o Hi, I have same error, while rendering image, did you solve it?
